libs/wingsdk/src/target-sim/container.inflight.ts

```diff
@@ -194,6 +241,114 @@ export class Container implements IContainerClient, ISimulatorResourceInstance {
     }
   }
 
+  private async docker(
+    command: string,
+    args: string[],
+    options: DockerSpawnOptions = {}
+  ): Promise<string> {
+    const child = this.dockerSpawn(command, args, options);
+    return child.join();
+  }
+
+  private dockerSpawn(
+    command: string,
+    args: string[],
+    options: DockerSpawnOptions = {}
+  ): DockerProcess {
+    let logStdout = options.stdout ?? false;
+    const logStderr = options.stderr ?? false;
+    const onError = options.onError;
+
+    // can be used to hide container logs (used in our end to end tests)
+    if (process.env.WING_HIDE_CONTAINER_LOGS) {
+      logStdout = false;
+    }
```

Comment:
Is this avoidable? Why do the logs need to be hidden in the e2e tests? Is there a way we can do filtering/redacting to make the output work fine in the e2e tests

Reply:
I am not sure it is avoidable because build/pull logs are not deterministic. They also have a bunch of timestamps and other non-deterministic things in them. Trying to be pragmatic about it...